.github/workflows/build.yml

```
@@ -0,0 +1,336 @@
+    outputs:
+      service_work_dir: ${{ steps.setenv.outputs.SERVICE_WORK_DIR }}
+      service_image_name: ${{ steps.setenv.outputs.SERVICE_IMAGE_NAME }}
+      service_dockerfile: ${{ steps.setenv.outputs.SERVICE_DOCKERFILE }}
+      db_folder_exists: ${{ steps.check-db-folder.outputs.folder_exists }}
+      db_work_dir: ${{ steps.setenv.outputs.DB_WORK_DIR }}
+      db_image_name: ${{ steps.setenv.outputs.DB_IMAGE_NAME }}
+      db_dockerfile: ${{ steps.setenv.outputs.DB_DOCKERFILE }}
+      tag: ${{ steps.tag.outputs.tag }}
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v3
```

actionlint (1.7.8)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)